.vscode/settings.json

```diff
@@ -1,15 +1,23 @@
 {
-    "python.linting.banditEnabled": true,
-    "python.linting.flake8Enabled": true,
-    "python.linting.pylintEnabled": true,
-    "python.linting.mypyEnabled": false,
-    "python.linting.pydocstyleEnabled": true,
-    "python.linting.enabled": true,
-    "python.linting.flake8Args": ["--ignore=E121,E402,F481,E501,F841", "--max-line-length=89"],
-    "python.formatting.provider": "black",
-    "python.testing.unittestEnabled": false,
-    "python.testing.pytestEnabled": true,
-    "python.testing.pytestArgs": [
-      "./tests/"
+  "python.linting.banditEnabled": true,
+  "python.linting.flake8Enabled": true,
+  "python.linting.pylintEnabled": true,
+  "python.linting.mypyEnabled": true,
+  "python.linting.pydocstyleEnabled": true,
+  "python.linting.enabled": true,
+  "python.linting.flake8Args": [
+    "--ignore=E121,E402,F481,E501,F841",
+    "--max-line-length=89"
```

Comment:
different settings to setup.cfg -> please check all

Reply:
done

Reply:
cant't you use the setup.cfg here?

Reply:
can't